mass_editing/__manifest__.py

```diff
@@ -2,21 +2,21 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
     "name": "Mass Editing",
-    "version": "13.0.1.1.0",
+    "version": "14.0.1.0.0",
     "author": "Serpent Consulting Services Pvt. Ltd., "
     "Tecnativa, "
     "GRAP, "
+    "Iván Todorovich, "
     "Odoo Community Association (OCA)",
     "category": "Tools",
     "website": "https://github.com/OCA/server-ux",
     "license": "AGPL-3",
     "summary": "Mass Editing",
-    "uninstall_hook": "uninstall_hook",
-    "depends": ["mass_operation_abstract"],
+    "depends": ["server_action_domain"],
```

Comment:
Can you avoid this dependency? We are removing one, but adding a new one, so the balance is similar. I would like to have this module again without any dependency.

Reply:
Could the functionality of server_action_domain be incorporated into
this one?

Reply:
Done! Dependency removed, `server_action_domain` module is here: https://github.com/OCA/server-ux/pull/257